lib/rules/jsx-max-props-per-line.js

```diff
@@ -24,8 +24,17 @@ module.exports = {
       type: 'object',
       properties: {
         maximum: {
-          type: 'integer',
-          minimum: 1
+          type: 'object',
+          properties: {
+            single: {
+              type: 'integer',
+              minimum: 1
+            },
+            multi: {
+              type: 'integer',
+              minimum: 1
+            }
+          }
```

Comment:
```suggestion
          },
          "additionalProperties": false
```
(if i'm remembering the syntax right)